src/style.css

```diff
@@ -46,11 +47,16 @@ button {
   width: 20em;
 }
 
-.all-recipes-display {
+.all-recipes-display,
```

Comment:
Let's get on a call at some point to talk about possible solutions - I think I have an idea to make this look the way we want it to